.release-it.json

```diff
@@ -0,0 +1,23 @@
+{
+    "hooks": {
+      "before:init": ["npm test"]
+    },
+    "git": {
+      "requireCleanWorkingDir": true,
+      "commitMessage": "chore: release ${version}",
+      "tagName": "${version}",
+      "push": true
+    },
+    "github": {
+      "release": true
+    },
+    "npm": {
+      "publish": true
+    },
+    "plugins": {
+      "@release-it/conventional-changelog": {
```

Comment:
If we're generating a CHANGELOG (which we do want), we should set the `preset` to `conventionalcommits`

See the [other example](https://github.com/Amsterdam/ee-docs/blob/develop/.release-it.json#L21):
```json
"@release-it/conventional-changelog": {
      "preset": "conventionalcommits",
      "infile": "CHANGELOG.md"
}
```